jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListBasicFiltersPresenter.java

```diff
@@ -192,9 +191,16 @@ protected void addSLAComplianceFilter() {
 
     @Override
     protected void onActiveFilterAdded(ActiveFilterItem activeFilterItem) {
-        if (activeFilterItem.getKey().equals(constants.State()) && activeFilterItem.getValue() instanceof List) {
+        if (activeFilterItem.getKey().equals(constants.State()) &&
+                activeFilterItem.getValue() instanceof List) {
             final List<String> values = (List<String>) activeFilterItem.getValue();
-            values.forEach(v -> view.checkSelectFilter(constants.State(), String.valueOf(v)));
+            values.forEach(v -> view.checkSelectFilter(constants.State(), v));
+        }
+
+        if (activeFilterItem.getKey().equals(COLUMN_ERROR_COUNT) &&
+                activeFilterItem.getValue() instanceof List) {
+            final List<String> values = (List<String>) activeFilterItem.getValue();
+            values.forEach(v -> view.checkSelectFilter(constants.Errors(), Boolean.valueOf(v.equals("0")).toString()));
```

Comment:
if you need to change the value, why not do it when you create the ActiveFilterItem as you already map coreFunctionFilter.getParameters()?

Reply:
error's ActiveFilterItem's parameters are "0" or "1",but selectInput's name's "true" and "false.". but state's ActiveFilterItem's parameters are "0" or "1"., it's selectInput's name also is "0","1"....
The logic in here :https://github.com/kiegroup/jbpm-wb/pull/1377/files#diff-d94e7092bb936da858898a7c5d50c2baR92